app/models/developer.rb

```diff
@@ -43,7 +44,7 @@ class Developer < ApplicationRecord
   validates :name, presence: true
   validates :response_rate, numericality: {greater_than_or_equal_to: 0, less_than_or_equal_to: 100}
 
-  pg_search_scope :filter_by_search_query, against: [:bio, :hero], using: {tsearch: {tsvector_column: :textsearchable_index_col}}
+  pg_search_scope :filter_by_search_query, against: [:bio, :hero], associated_against: {specialty: :name}, using: {tsearch: {tsvector_column: :textsearchable_index_col, prefix: true}}
```

Comment:
Nice! Do we need to update the index at all for this work?

Reply:
hi @joemasilotti  I dont think we need to update the index

Reply:
Excellent! Thanks for confirming.